src/components/task-plan/builder.cjsx

```diff
@@ -64,14 +71,14 @@ module.exports = React.createClass
     dueAt = @getQueriedDueAt()
     if dueAt? and moment(dueAt).isSame(opensAt, 'day')
       # make open date today if default due date is tomorrow
-      opensAt = moment(TimeStore.getNow()).format(TutorDateFormat)
+      opensAt = moment(TimeStore.getNow()).toDate()
 
```

Comment:
Since `TimeStore.getNow()` returns a date, what's gained from sending into `moment` and back to a date?